src/test/java/ca/sm360/cronitor/client/MonitorWithIdentificationTest.java

```diff
@@ -25,62 +26,64 @@ public void can_start_monitor_with_minimal_requirements() throws Exception {
 
         client.run(monitorCode);
 
-        verify(cronitorPinger).ping(new URL("https://cronitor.link/customMonitorCode/run?auth_key=customAuthKey"));
+        // verify(cronitorPinger).ping(Command.RUN.getValue(), "customMonitorCode", "customAuthKey", null);
```

Comment:
I'm getting an import error when trying to import `Command` and use it here like this. Any idea what I'm missing?